packages/feedback-service/src/schema.ts

```diff
@@ -0,0 +1,33 @@
+import { Document, Model, model, Schema } from 'mongoose';
+import { IFeedback } from "./types";
+
+export const FeedbackSchema: Schema = new Schema({
+  description: String,
+  experience: String,
+  iid: String,
+  module: { type: Schema.Types.ObjectId, ref: "Module" },
+  portalFeedback: Boolean,
+  timestamp: {
```

Comment:
@deshmukhmayur i think we decided to not define timestamps like this in one platform?
can u confirm?

Reply:
Yes. AIt should have a flat structure.
So `createdAt` `modifiedAt` should be a direct child of the schema.